src/Psr/Uri.php

```diff
@@ -10,16 +10,16 @@
 use Psr\Http\Message\UriInterface;
 
 /**
- * Class Uri
+ * Class URI
  *
  * This is an adapter class that implements the PSR-7 UriInterface that can't be implemented
- * by the Url class itself because it isn't designed to be immutable.
+ * by the URL class itself because it isn't designed to be immutable.
  */
 
 class Uri implements UriInterface
 {
     /**
-     * @var Url
+     * @var URL
```

Comment:
Refers to the Url class. Especially in phpdoc it's important that it's exactly cased like the class.

Reply:
Sorry, yeah. This was an oversight. Thanks